packages/fxa-profile-server/lib/img-workers.js

```diff
@@ -10,10 +10,23 @@ const AppError = require('./error');
 const config = require('./config');
 const logger = require('./logging')('img_workers');
 const request = require('./request');
+const https = require('https');
 
 const WORKER_URL = config.get('worker.url');
 const ACCEPT_ENCODING_HEADER = 'identity';
 
+exports.download = function download(url) {
+  return new P(function(resolve, reject) {
+    try {
+      https.get(url, (stream) => {
+        resolve(stream);
+      })
```

Comment:
Q: Is there any case where the `url` could be a vanilla-`http://` URL?
I did a hacky test and seem to get the following error if I try setting an HTTP `url` [in a standalone test.js script]:

```
TypeError [ERR_INVALID_PROTOCOL]: Protocol "http:" not supported. Expected "https:"
```